test/system/tests/createak.sh

```diff
@@ -31,14 +31,16 @@
 # THE POSSIBILITY OF SUCH DAMAGE.
 #;**********************************************************************;
 
+source helpers.sh
+
 onerror() {
     echo "$BASH_COMMAND on line ${BASH_LINENO[0]} failed: $?"
     exit 1
 }
 trap onerror ERR
 
 cleanup() {
-    rm -f ek.pub ak.pub ak.name ak.name ak.log
+    rm -f ek.pub ak.pub ak.name ak.name ak.out
```

Comment:
you are changing `.log` for `.out`. I don't really have a preference but I see the other tests use `.log` so I would just keep it for consistency.

Reply:
Agreed.